Dockerfile

```diff
@@ -1,10 +1,18 @@
 FROM uofa/apache2-php7:shepherd
 
+# Add the chromedriver repo using php, no wget or curl yet.
+RUN php -n -r '$x = file_get_contents("https://dl-ssl.google.com/linux/linux_signing_key.pub"); echo $x;' | apt-key add - \
+&& echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' > /etc/apt/sources.list.d/google-chrome.list
+
 # Upgrade all currently installed packages and install web server packages.
 RUN apt-get update \
-&& apt-get -y install php-sqlite3 php-xdebug php7.1-cli git wget sudo \
+&& apt-get -y install php-sqlite3 php-xdebug php7.1-cli git wget sudo unzip libnotify-bin google-chrome-stable \
 && sed -ri 's/^zend.assertions\s*=\s*-1/zend.assertions = 1/g' /etc/php/7.1/cli/php.ini \
 && sed -i 's/^\(allow_url_fopen\s*=\s*\).*$/\1on/g' /etc/php/7.1/mods-available/php_custom.ini \
+&& wget https://chromedriver.storage.googleapis.com/2.33/chromedriver_linux64.zip \
```

Comment:
https://gist.github.com/ziadoz/3e8ab7e944d02fe872c3454d17af31a5 has a nice thing for the chrome driver version so we don't need to maintain it - it'll just grab the latest.